crates/subspace-networking/src/behavior/persistent_parameters.rs

```diff
@@ -313,6 +314,7 @@ impl NetworkingParametersRegistry for NetworkingParametersManager {
             addresses,
             chrono::Duration::seconds(REMOVE_KNOWN_PEERS_GRACE_PERIOD_SECS),
             chrono::Duration::seconds(REMOVE_KNOWN_PEERS_GRACE_PERIOD_FOR_KADEMLIA_SECS),
+            //          &self.address_removed,
```

Comment:
I think this comment appeared during debugging and wasn't meant to be committed.